sonic_data_client/mixed_db_client.go

```diff
@@ -269,21 +435,14 @@ func (c *MixedDbClient) gnmiFullPath(prefix, path *gnmipb.Path) *gnmipb.Path {
 		origin = path.Origin
 	}
 	fullPath := &gnmipb.Path{Origin: origin}
-	if path.GetElement() != nil {
-		elements := path.GetElement()
-		if prefix != nil {
-			elements = append(prefix.GetElement(), elements...)
-		}
-		// Skip first elem
-		fullPath.Element = elements[1:]
-	}
 	if path.GetElem() != nil {
 		elems := path.GetElem()
 		if prefix != nil {
 			elems = append(prefix.GetElem(), elems...)
 		}
-		// Skip first elem
-		fullPath.Elem = elems[1:]
+		// Skip first two elem
+		// New GNMI path schema, /CONFIG_DB/localhost/PORT
```

Comment:
>localhost [](http://example.com/codeflow?start=38&length=9)

This is more general design change, not directly related to multiple database. Let's use a separate PR.

Reply:
Done